packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -123,7 +123,7 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
         <FocusZone>
           <table
             className={ css('ms-DatePicker-table', styles.table) }
-            role='grid'
+            role='application'
```

Comment:
@cschlechty @jspurlin 

`role=application` here; will this cause problems? Is this right?